# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/permission/PermissionUtils.cpp
# Commit: f932a2a98670
# Full Hash: f932a2a9867024a760dabaa611a0bbe6936475b5
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-06 05:18:37
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 3: Screen Wake Lock permission integration. r=webidl,dom-core,smaug,pbz,edgar
#   
#   Depends on D189509
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189510
# ==============================================================================

diff -r 231b0b4efd16 -r f932a2a98670 dom/permission/PermissionUtils.cpp
--- a/dom/permission/PermissionUtils.cpp	Tue Dec 05 23:58:06 2023 +0000
+++ b/dom/permission/PermissionUtils.cpp	Tue Dec 05 23:58:06 2023 +0000
@@ -19,7 +19,8 @@
     // "midi" is the only public permission but internally we have both "midi"
     // and "midi-sysex" (and yes, this is confusing).
     "midi"_ns,
-    "storage-access"_ns
+    "storage-access"_ns,
+    "screen-wake-lock"_ns
     // clang-format on
 };
 